common/modules/forseti/main.tf

```diff
@@ -0,0 +1,37 @@
+terraform {
+  backend "gcs" {}
+}
+
+variable "serviceaccount_key" {}
+variable "project_id" {}
+variable "infra_region" {}
+variable "auth_user_email" {}
+variable "organization_id" {}
+variable "domain_name" {}
+variable "server_grpc_allow_ranges" {}
+variable "cscc_source_id" {}
+variable "forseti_version" {}
+
+provider "google" {
+  credentials = "${var.serviceaccount_key}"
+  project     = "${var.project_id}"
+  region      = "${var.infra_region}"
+}
+
+module "forseti" {
+  source  = "terraform-google-modules/forseti/google"
```

Comment:
I would not worry about these modules being downloaded every time CI runs, but I already switched to local copy to avoid situations similar to what we see with image tags.